test/built-ins/Proxy/get/report-same-value-for-non-writable-non-configurable-property.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.8
+description: >
+    [[Get]] (P, Receiver)
+
+    proxy must report the same value for a non-writable, non-configurable
+    property
+info: >
+    13. If targetDesc is not undefined, then
+        a. If IsDataDescriptor(targetDesc) and targetDesc.[[Configurable]] is
+        false and targetDesc.[[Writable]] is false, then
+            i. If SameValue(trapResult, targetDesc.[[Value]]) is false, throw a
+            TypeError exception.
+---*/
+
+var target = {};
+var p = new Proxy(target, {
+    get: function() {
+        return 2;
```

Comment:
The file for this test is inaccurate because the test body asserte the behavior when the result is *not* the same value. I think you should keep it (but rename it) and include another test that asserts no error when the trap does return the same value.